projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts

```diff
@@ -16,13 +17,23 @@ import {map} from 'rxjs/operators';
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class TuiFontSizeComponent {
+    readonly normalFontSize = 15;
```

Comment:
Maybe move it to a constant or DI token? Overall, these font options don't seem to be customizable, right?

Reply:
fixed